api/server.go

```diff
@@ -230,6 +231,9 @@ func RunServer(dry bool) http.Handler {
 				fatal(err)
 			}
 		}
+		if startupMessageProvisioner, ok := app.Provisioner.(provision.StartupMessageProvisioner); ok {
+			startupMessageProvisioner.FprintStartupMessage(os.Stdout)
```

Comment:
Please change this to simply return a string. The caller then can use `fmt.Printf` for now. If we start using some logging library in the future this would cause less changes.